user_modules/dummy_connecttest.lua

```diff
@@ -405,7 +406,10 @@ function module:initHMI_onReady(hmi_table)
           ["mandatory"] = hmi_table_element.mandatory,
           ["params"] = hmi_table_element.params
         })
-        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", hmi_table_element.params)
+        local function sendHMIResponse ()
```

Comment:
@VjKlepikov Please remove redundant whitespace 
```suggestion
        local function sendHMIResponse()
```